### DISCUSSION TITLE
Objenesis is interested in joining

### DISCUSSION THREAD

**Discussion by henri-tremblay** [commonhaus/foundation#142] [Joining Commonhaus]
### Project information

- **Project name**: Objenesis
- **Project website**: https://objenesis.org/
- **Code repository**: https://github.com/easymock/objenesis
- **License**: Apache 2.0

### Describe your project

Objenesis is written in Java and widely used to create objects **without calling a constructor**. Quite useful for mocking frameworks and frameworks doing dynamic proxies. For this reason, it is used by EasyMock, Mockito and Spring. It has close to 30 million downloads per month.

**Comment by ebullient** [comment]
Still working on making this cleaner and other bootstrapping, but here is what we need to move forward. It may feel weird to sign the agreement twice (for both EasyMock and  Objenesis). I think it's fine if combine them in one PR, but I do think (despite the noise) that it should be one document for each so future us can understand what happened. ;)

- [ ] **Review the bylaws**, especially sections about [projects](https://www.commonhaus.org/bylaws/cf-membership.html#projects), [decision making](https://www.commonhaus.org/bylaws/decision-making.html), and [voting for significant decisions](https://www.commonhaus.org/bylaws/cf-council.html#voting). You may have seen some PRs go through as I work with the current EGC to create a record of approval with the newly established EGC) to ratify them and you're welcome to weigh in on these.
- [ ] **Read the [bootstrapping agreement](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md)**.
- [ ] If all is well, and you agree to participate in the bootstrapping phase to help us build this thing, then **create a PR** with the following:

    1. A filled out copy of the bootstrapping agreement. You can make a copy of [bootstrapping.md](https://github.com/commonhaus/foundation/blob/main/agreements/bootstrapping/bootstrapping.md) file and check that in, or checkin a filled out [pdf](https://github.com/commonhaus/foundation/releases/download/final-draft/bootstrapping.pdf). Please follow naming conventions.
    2. An updated [CONTACTS.yaml](https://github.com/commonhaus/foundation/blob/main/CONTACTS.yaml) that names your project representative in the EGC section
    3. An updated [PROJECTS.yaml](https://github.com/commonhaus/foundation/blob/main/PROJECTS.yaml) that provides details / describes your project
    4. An updated [TRADEMARKS.md](https://github.com/commonhaus/foundation/blob/main/TRADEMARKS.md) that adds your project name and project logo (if you have one). Please include the footnote reference, as we are using your project name and logo with your permission until we have a formal agreement in place and resources are transferred.

    The commit should be "Signed-off-by" your GH login (`git commit -s -m "<project> bootstrapping agreement, yyyy-mm-dd"`)

**Reply by henri-tremblay** [reply]
#156 created

**Comment by henri-tremblay** [comment]
So this one is done. I guess we can close the discussion.

**Comment by ebullient** [comment]
Resolved by #156 

Welcome to Commonhaus! 🎉 💖 🚀
